docs/serving/services/creating-services.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Creating Knative Services
+type: “docs”
+---
+
+Knative services are used to deploy an application. Knative Services attempt to model a more logical definition of a Service or Application. To achieve this, Services use the concept of Revisions which are each backed by a deployment with 2 Kubernetes Services.   
```

Comment:
I'm not sure what you mean by "attempt to model a more logical definition of a Service or Application." I would focus on how Knative simplifies managing serverless apps, or the other benefits, like scale-to-zero.

Reply:
For the rest of this section, I'd be a little more direct. Something like, "Knative Services are made up of a Route and a Configuration. Every time the Configuration is updated, a new Revision is created."

I'd also move the link to the K8s doc to the end of the paragraph.